samples/django/google_user/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==1.10.0
+oauth2client==3.0.0
```

Comment:
I think this will not work out-of-the-box till https://github.com/google/oauth2client/issues/593 is released